iquip/apps/monitor.py

```diff
@@ -116,3 +120,149 @@ def _setOverrideButtonText(self, override: bool):
             self.overrideButton.setText("Overriding")
         else:
             self.overrideButton.setText("Not Overriding")
+
+
+class _TTLOverrideThread(QThread):
+    """QThread for setting the override of all TTL channels through the proxy server.
+    
+    Attributes:
+        override: Override value to set.
+        ip: Proxy server IP address.
+        port: Proxy server PORT number.
+    """
+
+    def __init__(self, override: bool, ip: str, port: int, parent: Optional[QObject] = None):
+        """Extended.
+        
+        Args:
+            override, ip, port: See the attributes section.
+        """
+        super().__init__(parent=parent)
+        self.override = override
+        self.ip = ip
+        self.port = port
+
+    def run(self):
+        """Overridden.
+        
+        Sets the override of all TTL channels through the proxy server.
+
+        Since it just sends a POST query, it cannot be guaranteed that
+        the override will be applied immediately.
+        """
+        params = {"value": self.override}
+        try:
+            response = requests.post(
+                f"http://{self.ip}:{self.port}/ttl/override/",
+                params=params,
+                timeout=10
+            )
+            response.raise_for_status()
+        except requests.exceptions.RequestException:
+            logger.exception("Failed to set the override of all TTL channels.")
+            return
+
+
+class _TTLLevelThread(QThread):
+    """QThread for setting the level of the target TTL channel through the proxy server.
+    
+    Attributes:
+        channel: Target TTL channel number.
+        level: Level value to set.
+        ip: Proxy server IP address.
+        port: Proxy server PORT number.
+    """
+
+    def __init__(
+        self,
+        channel: int,
+        level: bool,
+        ip: str,
+        port: int,
+        parent: Optional[QObject] = None
+    ):  # pylint: disable=too-many-arguments
+        """Extended.
+        
+        Args:
+            target, level, ip, port: See the attributes section.
+        """
+        super().__init__(parent=parent)
+        self.channel = channel
+        self.level = level
+        self.ip = ip
+        self.port = port
+
+    def run(self):
+        """Overridden.
+        
+        Sets the level of the target TTL channel through the proxy server.
+
+        Since it just sends a POST query, it cannot be guaranteed that
+        the level will be applied immediately.
```

Comment:
Doesn't POST send a response as well? I think we cannot say the reason is POST.

Reply:
You're right! The correct reason is that the proxy server doesn't wait until applying it is done.

Reply:
I will remove the reason because it is responsible for the proxy server, not IQUIP.